### DISCUSSION TITLE
ViGEmBus causing crash when resuming from hibernate mode

### DISCUSSION THREAD

**Discussion by mastertofu** [nefarius/ViGEmBus#171] [Support]
Hello,

The issue happens when the computer is resuming from hibernation mode. Crashes right in the Windows loading screen and reboots.
Hibernation is vital for me and ViGEmBus is required by Sunshine https://github.com/LizardByte/Sunshine

I have collected several minidumps, and have an extract of my current specs.
please let me know where to upload. :)

Thank you!

**Comment by nefarius** [comment]
Driver version? OS build? Architecture?

**Reply by mastertofu** [reply]
Hi!
Thank you for picking this up!

Driver version: 1.21.442 from the pre-built releases https://github.com/ViGEm/ViGEmBus/releases/tag/v1.21.442.0
OS Build: Win 11 22H2 (22621.1344)
Arch: x64

I have minidumps and full spec extract from dxdiag.
Let me know if it is safe to upload here or if you prefer to have it uploaded somewhere.

thanks!

**Reply by nefarius** [reply]
You can upload them here: files dot nefarius dot at, no need to tell me the resulting links, just tell me when done so.

**Reply by mastertofu** [reply]
done!
filename: bsod_from_hibernate.zip

Thank you!!!!!

**Reply by nefarius** [reply]
Received!

**Reply by mastertofu** [reply]
cheers!
Please let me know if you need more info, or how I can help!
